CodeEdit/Features/InspectorArea/HistoryInspector/HistoryInspectorView.swift

```diff
@@ -43,11 +43,13 @@ struct HistoryInspectorView: View {
             }
         }
         .onReceive(editorManager.activeEditor.objectWillChange) { _ in
-            model.setFile(url: editorManager.activeEditor.selectedTab?.url.path)
+            Task {
+                await model.setFile(url: editorManager.activeEditor.selectedTab?.url.path)
+            }
         }
```

Comment:
Needs to add these listeners to receive updates for split editors. It's ugly but required for now.
```suggestion
        }
        .onChange(of: editorManager.activeEditor) { _ in
            Task {
                await model.setFile(url: editorManager.activeEditor.selectedTab?.url.path)
            }
        }
        .onChange(of: editorManager.activeEditor.selectedTab) { _ in
            Task {
                await model.setFile(url: editorManager.activeEditor.selectedTab?.url.path)
            }
        }
```